Framework/Core/System/Xml/XmlWriter.cs

```diff
@@ -802,7 +770,7 @@ public void WriteStartAttribute(string prefix, string localName, string ns)
 
             string localPrefix = "";
 
-            if(prefix != null && prefix.Length == 0) prefix = null;
+            if (prefix != null && prefix.Length == 0) prefix = null;

```

Comment:
Suggestion: While you're touching these lines, add braces and move the block to its own line.